json.go

```diff
@@ -0,0 +1,36 @@
+package opslevel
+
+import (
+	"encoding/json"
+	"strconv"
+)
+
+// JSON is a specialized map[string]string to support proper graphql serialization
+type JSON map[string]string
```

Comment:
This is what the HEADERS field uses and AFAICT its a new type we've never had in the public API before.